src/module/models/shared.ts

```diff
@@ -306,16 +306,16 @@ export class SyncUUIDRefField extends fields.StringField {
 // Use this to represent a field that is effectively just a number, but should present as a min/max/value field in expanded `system` data
 // This is 10% so we can show them with bars, and 90% because usually the max is computed and we don't want to confuse anyone
 export class FakeBoundedNumberField extends fields.NumberField {
-  constructor(a: any = undefined) {
-    super(a);
+  constructor(options: object = {}) {
+    super(options);
   }
 
   /** @override */
   initialize(value: string, model: any) {
```

Comment:
Just noticed - wouldn't `value` be a `number`?

Reply:
Yeah probably - these aren't really type checked anyways though, since the `fields` api is entirely novel to v10+ and my attempts to shim it just didn't really end up working. I'll do a lil fix though. Definitely just copy pasted something.